python/ray/rllib/example.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
```

Comment:
Do we want to include this line? E.g., we can just have people run it with `python example.py` and not make it executable?